### PR TITLE
ci, store/util: try to capture clock jump

### PR DESCRIPTION
**Pull request by overvenus**
Cc #3044

### CODE REVIEW

File: src/raftstore/store/util.rs [hunk 1]
```diff
@@ -427,6 +427,15 @@ mod tests {
 
     #[test]
     fn test_lease() {
+        #[inline]
+        fn sleep_test(duration: TimeDuration, lease: &Lease, state: LeaseState) {
```
Comment (hicqu) [inline src/raftstore/store/util.rs]: I think it could still panic on assert fail.

**Comment by hicqu** [comment]
LGTM.

File: .circleci/config.yml [hunk 2]
```diff
@@ -23,16 +23,18 @@ jobs:
             echo 'export CPPFLAGS=-I$HOME/.local/include' >> $BASH_ENV
             echo 'export CXXFLAGS=-I$HOME/.local/include' >> $BASH_ENV
             echo 'export LOCAL_PREFIX=$HOME/.local' >> $BASH_ENV
+            cat /boot/config-`uname -r`
+            dmesg
```
Comment (BusyJay) [inline .circleci/config.yml]: Why?

Comment (overvenus) [reply .circleci/config.yml]: For printing kernel config and clock source.

**Pull request merged**